controller/work-item-link-type-blackbox_test.go

```diff
@@ -136,23 +139,24 @@ func (s *workItemLinkTypeSuite) TearDownTest() {
 
 // createDemoType creates a demo work item link type of type "name"
 func (s *workItemLinkTypeSuite) createDemoLinkType(name string) *app.CreateWorkItemLinkTypePayload {
-	//	//   1. Create at least one work item type
-	//	workItemTypePayload := CreateWorkItemType("foo.bug")
-	//	_, workItemType := test.CreateWorkitemtypeCreated(s.T(), nil, nil, s.typeCtrl, workItemTypePayload)
-	//	require.NotNil(s.T(), workItemType)
+	//   1. Create a space
+	createSpacePayload := CreateSpacePayload("test-space", "description")
+	_, space := test.CreateSpaceCreated(s.T(), s.svcSpace.Context, s.svcSpace, s.spaceCtrl, createSpacePayload)
+	s.spaceID = space.Data.ID
+	require.NotNil(s.T(), space)
```

Comment:
Hehehe good one :)! I don't insist I just simply followed the code structured as done in many other places.